components/engine_rocks/src/rocks_metrics.rs

```diff
@@ -1588,31 +1524,6 @@ lazy_static! {
         "Histogram of WAL file sync micros",
         &["db", "type"]
     ).unwrap();
-    pub static ref STORE_ENGINE_STALL_L0_SLOWDOWN_COUNT_VEC: GaugeVec = register_gauge_vec!(
-        "tikv_engine_stall_l0_slowdown_count",
-        "Histogram of stall l0 slowdown count",
-        &["db", "type"]
-    ).unwrap();
-    pub static ref STORE_ENGINE_STALL_MEMTABLE_COMPACTION_COUNT_VEC: GaugeVec = register_gauge_vec!(
-        "tikv_engine_stall_memtable_compaction_count",
-        "Histogram of stall memtable compaction count",
-        &["db", "type"]
-    ).unwrap();
-    pub static ref STORE_ENGINE_STALL_L0_NUM_FILES_COUNT_VEC: GaugeVec = register_gauge_vec!(
-        "tikv_engine_stall_l0_num_files_count",
-        "Histogram of stall l0 num files count",
-        &["db", "type"]
-    ).unwrap();
-    pub static ref STORE_ENGINE_HARD_RATE_LIMIT_DELAY_VEC: GaugeVec = register_gauge_vec!(
-        "tikv_engine_hard_rate_limit_delay_count",
-        "Histogram of hard rate limit delay count",
-        &["db", "type"]
-    ).unwrap();
-    pub static ref STORE_ENGINE_SOFT_RATE_LIMIT_DELAY_VEC: GaugeVec = register_gauge_vec!(
-        "tikv_engine_soft_rate_limit_delay_count",
-        "Histogram of soft rate limit delay count",
-        &["db", "type"]
-    ).unwrap();
```

Comment:
Clean up in tikv_details as well?

Reply:
I don't see them in tikv_details. Can you point me to the place?